File: src/test/java/day01/TrebuchetTest.java
package day01;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class TrebuchetTest {

    @Test
    void firstStar() {
        Trebuchet trebuchet = new Trebuchet("day01/input1.txt");
        assertEquals(142, trebuchet.firstStar());
    }

    @Test
    void secondStar() {
        Trebuchet trebuchet = new Trebuchet("day01/input2.txt");
        assertEquals(281, trebuchet.secondStar());
    }
}

File: src/test/kotlin/day01/TrebuchetTest.kt
package day01

import org.junit.jupiter.api.Assertions.*
import org.junit.jupiter.api.Test

internal class TrebuchetTest {
    @Test
    fun firstStar() {
        assertEquals(142, Trebuchet("day01/input1.txt").firstStar())
    }

    @Test
    fun secondStar() {
        assertEquals(281, Trebuchet("day01/input2.txt").secondStar())
    }
}